generated/nidaqmx/_grpc_interpreter.py

```diff
@@ -0,0 +1,2757 @@
+        )
+
+    def create_di_chan(
+            self, task, lines, name_to_assign_to_lines, line_grouping):
+        response = self._invoke(
+            self._client.CreateDIChan,
+            grpc_types.CreateDIChanRequest(line_grouping_raw=line_grouping, lines=lines, name_to_assign_to_lines=name_to_assign_to_lines, task=task),
+        )
+
+    def create_do_chan(
+            self, task, lines, name_to_assign_to_lines, line_grouping):
+        response = self._invoke(
+            self._client.CreateDOChan,
+            grpc_types.CreateDOChanRequest(line_grouping_raw=line_grouping, lines=lines, name_to_assign_to_lines=name_to_assign_to_lines, task=task),
+        )
+
+    def create_lin_scale(
+            self, name, slope, y_intercept, pre_scaled_units, scaled_units):
+        response = self._invoke(
+            self._client.CreateLinScale,
+            grpc_types.CreateLinScaleRequest(name=name, pre_scaled_units_raw=pre_scaled_units, scaled_units=scaled_units, slope=slope, y_intercept=y_intercept),
+        )
+
+    def create_map_scale(
+            self, name, prescaled_min, prescaled_max, scaled_min, scaled_max,
+            pre_scaled_units, scaled_units):
+        response = self._invoke(
+            self._client.CreateMapScale,
+            grpc_types.CreateMapScaleRequest(name=name, pre_scaled_units_raw=pre_scaled_units, prescaled_max=prescaled_max, prescaled_min=prescaled_min, scaled_max=scaled_max, scaled_min=scaled_min, scaled_units=scaled_units),
+        )
+
+    def create_polynomial_scale(
+            self, name, forward_coeffs, num_forward_coeffs_in, reverse_coeffs,
+            num_reverse_coeffs_in, pre_scaled_units, scaled_units):
+        response = self._invoke(
+            self._client.CreatePolynomialScale,
+            grpc_types.CreatePolynomialScaleRequest(forward_coeffs=forward_coeffs, name=name, num_forward_coeffs_in=num_forward_coeffs_in, num_reverse_coeffs_in=num_reverse_coeffs_in, pre_scaled_units_raw=pre_scaled_units, reverse_coeffs=reverse_coeffs, scaled_units=scaled_units),
+        )
+
+    def create_table_scale(
+            self, name, prescaled_vals, num_prescaled_vals_in, scaled_vals,
+            num_scaled_vals_in, pre_scaled_units, scaled_units):
+        response = self._invoke(
+            self._client.CreateTableScale,
+            grpc_types.CreateTableScaleRequest(name=name, num_prescaled_vals_in=num_prescaled_vals_in, num_scaled_vals_in=num_scaled_vals_in, pre_scaled_units_raw=pre_scaled_units, prescaled_vals=prescaled_vals, scaled_units=scaled_units, scaled_vals=scaled_vals),
+        )
+
+    def create_task(self, session_name):
+        response = self._invoke(
+            self._client.CreateTask,
+            grpc_types.CreateTaskRequest(session_name=session_name),
+        )
+        return response.task
```

Comment:
This will be taken care of in a separate PR as part of [Task 2373857](https://dev.azure.com/ni/DevCentral/_workitems/edit/2373857): Update task class to handle initialization methods